### ISSUE TITLE
Table editing, creating, etc and screen readers

### ISSUE THREAD

**Issue by atulwy** [webyog/sqlyog-community#1476]
Original [issue 1476](https://code.google.com/p/sqlyog/issues/detail?id=1476) created by webyog on 2010-09-16T18:04:22.000Z:

When someones tries to use your product with an screen reader (e.k. jaws, nvda), he'll fail when he tries to for example alter a table. The focus lands on the fields table that's perhaps desired behavior, but that's the end. If he wants to for example check/uncheck the not null? checkbox or get to the advanced settings button, he must use weird and tricky methods which are dependent on the screen reader he/she is using. If you could do anythink with this situation (for example tree structure as in user manager), a lot of users would be happy.
